examples/react-design-system/src/builder-menus.js

```diff
@@ -0,0 +1,85 @@
+export const menus = [
```

Comment:
why this change? i try to remove non-critical abstractions from examples so people can more clearly see things right inline and not have to chase down references and understand how used. it also allows easier type completion to leave as it was

`Builder.register('insertMenu', { ... }`

Reply:
left over from a former config where it required the menus to be passed down, I'll revert